modules/op-using-custom-scc-to-improve-prod-security.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * cicd/pipelines/running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines.adoc
+:_content-type: PROCEDURE
+
+[id="op-using-custom-scc-to-improve-prod-security_{context}"]
+= Using a custom SCC to improve pod security
+
+To improve the security of the builds and runs, cluster adminsitrators can use custom security context constraints (SCC).
+
+.Procedure
+
+* Using the following example as a model, create a `pipelines-scc` SCC and configure its `runAsUser` and `seLinuxContext` fields.
++
+[source,yaml,subs="attributes+"]
+----
+...
+runAsUser:
+  type: MustRunAs
+  uid: 1000
+seLinuxContext:
+  type: MustRunAs
+...
+----
++
+[NOTE]
+====
+The `pipeline` service account allows images to run only as user `1000`, not as root.
+====
+
+.Example: Modified SCC configuration
+[source,yaml]
+----
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: true
+allowPrivilegedContainer: false
+allowedCapabilities: null
+apiVersion: security.openshift.io/v1
+defaultAddCapabilities: null
+fsGroup:
+  type: MustRunAs
+groups:
+- system:cluster-admins
+kind: SecurityContextConstraints
+metadata:
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    kubectl.kubernetes.io/last-applied-configuration: |
```

Comment:
Do we need this annotation and generated fields such as creationTimestamp, generation and ownerReference?

Reply:
@chmouel Need your inputs.

Reply:
we don't need them indeed...